Helper/Order.php

```diff
@@ -726,9 +726,6 @@ public function saveUpdateOrder($reference, $storeId = null, $boltTraceId = null
         $parentQuote = $this->cartHelper->getQuoteById($parentQuoteId);
 
         if ($parentQuote) {
-            if ($order->getState() === OrderModel::STATE_PENDING_PAYMENT) {
```

Comment:
@vitaliyreznikov this is moved into hook handler (updateOrderPayment). It affects your work on checkboxes.